lib/src/components/NxVulnerabilityIndicatorLegend/NxVulnerabilityIndicatorLegend.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {
+  NxVulnerabilityIndicatorLegendProps,
+  nxVulnerabilityIndicatorLegendPropTypes,
+  VulnerabilityProps
+} from './types';
+import NxVulnerabilityIndicator from '../NxVulnerabilityIndicator/NxVulnerabilityIndicator';
+import './NxVulnerabilityIndicatorLegend.scss';
+
+const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const VulnerabilityIndicator = ({ severityRating }: VulnerabilityProps) => {
+  return (
+    <div className="nx-vulnerability-indicator-legend__vulnerability-container">
+      <NxVulnerabilityIndicator severityRating={severityRating} />
+      <span>{capitalize(severityRating)}</span>
+    </div>
+  );
+};
+
+const NxVulnerabilityIndicatorLegend = forwardRef<HTMLDivElement, NxVulnerabilityIndicatorLegendProps>(
+    (props: NxVulnerabilityIndicatorLegendProps, ref) => {
+      const {
+        critical,
+        high,
+        medium,
+        low,
+        none,
+        vertical,
+        header,
+        className,
+        ...attrs
+      } = props;
```

Comment:
One line

Reply:
Fixed in: [39456a8](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/39456a8c538c2cc72dde789504f5a8bb8340e3b1)